tests/unit/test.program.js

```diff
@@ -329,15 +331,23 @@ describe('program.main', () => {
 
 
 describe('program.defaultVersionGetter', () => {
+  let root = path.join(__dirname, '..', '..');
 
-  it('returns the package version', () => {
-    let root = path.join(__dirname, '..', '..');
+  it('returns the package version in production', () => {
     let pkgFile = path.join(root, 'package.json');
     return fs.readFile(pkgFile)
       .then((pkgData) => {
-        assert.equal(defaultVersionGetter(root),
-                     JSON.parse(pkgData).version);
+        const testBuildEnv = {localEnv: 'production'};
+        assert.equal(defaultVersionGetter(root, testBuildEnv),
```

Comment:
You could make this a bit more concise like:
````
assert.equal(defaultVersionGetter(root, {localEnv: 'production'})
````
Not a big deal though, don't worry about changing it.